IMDSSample.pl

```diff
@@ -10,7 +10,7 @@
 my $md_url = 'http://169.254.169.254/metadata/instance?api-version=2019-03-11';
 my $headers = { Metadata => 'true' };
 
-my $response = HTTP::Tiny->new->get($md_url, { headers => $headers });
+my $response = HTTP::Tiny->new(proxy => undef)->get($md_url, { headers => $headers });
```

Comment:
We might want to consider duplicating our proxy disclaimer as a comment in each example.